src/NuGet.Core/NuGet.Protocol/Plugins/PluginDiscoverer.cs

```diff
@@ -17,25 +19,35 @@ public sealed class PluginDiscoverer : IPluginDiscoverer
     {
         private bool _isDisposed;
         private List<PluginFile> _pluginFiles;
-        private readonly string _rawPluginPaths;
+        private readonly string _envVariableNetCoreAndNetFXPluginPaths;
```

Comment:
I was wondering if there are better descriptive names that aren't as wordy
 Some ideas, _envVariableNetCoreAndNetFXPluginPaths could be `envVariablePlatformSpecificPluginPaths`